src/components/memberGrid.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const MemberGrid = ({ data }) => {
+  return (
+    <div className="table">
+      {data.length > 0 && data.map((member) => <RowElement data={member} />)}
+    </div>
+  );
+};
+
+const RowElement = ({ data }) => {
+  return <div className="row">
+    <img src={data.} alt="img" />
```

Comment:
Do not leave src value undefined....for now, use a placeholder image

Reply:
This has been solved. It  was left due to lack of backend api.